cico_setup.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+# Output command before executing
+set -x
+
+# Exit on error
+set -e
+
+# Source environment variables of the jenkins slave
+# that might interest this worker.
+function load_jenkins_vars() {
+  if [ -e "jenkins-env" ]; then
+    cat jenkins-env \
+      | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+      | sed 's/^/export /g' \
+      > ~/.jenkins-env
+    source ~/.jenkins-env
+  fi
+}
+
+function install_deps() {
+  # We need to disable selinux for now, XXX
+  /usr/sbin/setenforce 0
+
+  # Get all the deps in
+  yum -y install \
+    docker \
+    make \
+    git \
+    curl
+
+  service docker start
+
+  echo 'CICO: Dependencies installed'
+}
+
+function cleanup_env {
+  EXIT_CODE=$?
+  echo "CICO: Cleanup environment: Tear down test environment"
+  make integration-test-env-tear-down
+  echo "CICO: Exiting with $EXIT_CODE"
+}
+
+function prepare() {
+  # Let's test
+  make docker-start
+  make docker-check-go-format
+  make docker-deps
+  #make docker-analyze-go-code
+  make docker-generate
+  make docker-build
+  echo 'CICO: Preparation complete'
+}
+
+function run_tests_without_coverage() {
+  make docker-test-unit-no-coverage-junit
+  #make integration-test-env-prepare
+  #trap cleanup_env EXIT
+
+  # Check that postgresql container is healthy
+  #check_postgres_healthiness
+
+  #make docker-test-migration
+  #make docker-test-integration-no-coverage
```

Comment:
hehe, the notification services doesn't have a database backend, all tests run as unit-test :)